src/mds/MDLog.cc

```diff
@@ -170,7 +170,37 @@ uint64_t MDLog::get_safe_pos() const
   return journaler->get_write_safe_pos(); 
 }
 
+// estimate the replay completion time based on mdlog journal pointers
+EstimatedReplayTime MDLog::get_estimated_replay_finish_time() {
+  ceph_assert(mds->is_replay());
 
+  EstimatedReplayTime estimated_time(0, std::chrono::seconds::zero(), std::chrono::seconds::zero());
```

Comment:
```suggestion
  EstimatedReplayTime estimated_time{0, std::chrono::seconds::zero(), std::chrono::seconds::zero()};
```

causes

```
/home/jenkins-build/build/workspace/ceph-api/src/mds/MDLog.cc:177:23: error: no matching constructor for initialization of 'EstimatedReplayTime'
  EstimatedReplayTime estimated_time(0, std::chrono::seconds::zero(), std::chrono::seconds::zero());
                      ^              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/jenkins-build/build/workspace/ceph-api/src/mds/mdstypes.h:1032:8: note: candidate constructor (the implicit copy constructor) not viable: requires 1 argument, but 3 were provided
struct EstimatedReplayTime {
       ^
```

Reply:
Nice catch - fixing.

Reply:
hmm. I did compile this change before posting.